File: JW/CodeTest/OT_002.py
def solution(arr):
    answer = []
    for x in range(len(arr)):
        if x == 0 :
            temp_num = arr[x]
            answer.append(arr[x])
        else:
            if temp_num != arr[x]:
                answer.append(arr[x])
                temp_num = arr[x]
    
    
    return answer

File: YJ/OT/OT_002.js
function solution(arr)
{
    let answer = [];
    let value = arr[0];
    
    answer.push(value);
    
    for (let i = 1; arr.length > i; i++) {
        if(value != arr[i]){
            answer.push(arr[i]);
            value = arr[i];
        }   
    }
    
    return answer;
}